packages/react/src/components/QuilttButton.tsx

```diff
@@ -11,7 +11,13 @@ type QuilttButtonProps<T extends AnyTag> = PropsWithChildren<
     connectorId: string
     connectionId?: string // For Reconnect Mode
     institution?: string // For Connect Mode
-  } & ConnectorSDKCallbacks
+    // Override the native onClick handler
+    onClick?: (event: MouseEvent<HTMLElement>) => void
+  } & Omit<ConnectorSDKCallbacks, 'onLoad'> & {
+      // Separate the SDK onLoad from the HTML onLoad
+      onLoad?: ConnectorSDKCallbacks['onLoad']
+      onHtmlLoad?: React.ReactEventHandler<HTMLElement>
+    }
```

Comment:
1. can you break this down a bit into separate types? having a hard time following all this. 
2. Also what's the SDK onLoad? Is the callback not registering because `onLoad` is reserved?

Reply:
1. Yes, I'll get that in
2. No. It's the other way around, callbacks are registering for the SDK. But it's being mangled together with the HTML onLoad IF it exists